.eslintrc.json

```diff
@@ -8,6 +8,7 @@
   "extends": "airbnb",
   "parser": "babel-eslint",
   "rules": {
-    "camelcase": 0
+    "camelcase": 0,
+    "object-shorthand": [2, "properties"]
```

Comment:
Actually, after [my first refactor](https://github.com/artsy/metaphysics/pull/373/commits/b495d016c9b372307d2ec33d868545f1fb7775cb), I ended up [no longer needing this](https://github.com/artsy/metaphysics/pull/373/commits/deb1d5c5924dbdb8c8697de38505c00020d2b94c) anyways, because I got rid of the shared `fetch` method.